.github/workflows/jenkins-security-scan.yml

```
@@ -0,0 +1,21 @@
+name: Jenkins Security Scan
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [ opened, synchronize, reopened ]
+  workflow_dispatch:
+
+permissions:
+  security-events: write
+  contents: read
+  actions: read
+
+jobs:
+  security-scan:
+    uses: jenkins-infra/jenkins-security-scan/.github/workflows/jenkins-security-scan.yaml@v2
+    with:
+      java-cache: 'maven' # Optionally enable use of a build dependency cache. Specify 'maven' or 'gradle' as appropriate.
+      java-version: 11 # Optionally specify what version of Java to set up for the build, or remove to use a recent default.
```

we're now on Java 17 🙃 
```suggestion
      java-version: 17 # Optionally specify what version of Java to set up for the build, or remove to use a recent default.
```